packages/iTwinUI-react/src/core/Alert/Alert.test.tsx

```diff
@@ -63,24 +61,27 @@ it('renders sticky alert correctly', () => {
     <Alert isSticky>This is sticky alert.</Alert>,
   );
 
-  const alert = container.querySelector(
-    '.iui-alert-informational.iui-sticky',
-  ) as HTMLElement;
+  const alert = container.querySelector('.iui-alert') as HTMLElement;
   expect(alert).toBeTruthy();
+  expect(alert).toHaveAttribute('data-iui-variant', 'sticky');
   getByText('This is sticky alert.');
 });
 
-(['informational', 'positive', 'warning', 'negative'] as Array<
-  'informational' | 'positive' | 'warning' | 'negative'
->).forEach((type) => {
+(
+  ['informational', 'positive', 'warning', 'negative'] as Array<
+    'informational' | 'positive' | 'warning' | 'negative'
+  >
+).forEach((type) => {
```

Comment:
optional: Can revert this formatting change. Or maybe better to keep it since it is a small PR, so better to do formatting changes along with this PR.

Reply:
Every new PR seems to have formatting issues. I think this could potentially be because the `website` uses a different version of `prettier` which is taking preference. Feel free to look into it if you have time